examples/mms-create.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+require_once(__DIR__ . '/../autoload.php');
+
+$MessageBird = new \MessageBird\Client('YOUR_ACCESS_KEY'); // Set your own API access key here.
+
+$MmsMessage = new \MessageBird\Objects\MmsMessage();
+$MmsMessage->originator = 'MessageBird';
+$MmsMessage->recipients = array(31621938645);
+$MmsMessage->subject = "Check out this cool MMS";
+$MmsMessage->body = 'Have you seen this logo?';
+$MmsMessage->mediaUrls = array('https://www.messagebird.com/assets/images/og/messagebird.gif');
+
+try {
+    $MmsMessageResult = $MessageBird->mmsMessages->create($MmsMessage);
+    var_dump($MmsMessageResult);
+
```

Comment:
Lets remove all those unnecessary empty lines in all example files

Reply:
Done